mono/metadata/object.c

```diff
@@ -5271,20 +5293,24 @@ mono_string_new_len (MonoDomain *domain, const char *text, guint length)
 {
 	MONO_REQ_GC_UNSAFE_MODE;
 
+	MonoError merror;
```

Comment:
I see the issue, there's a conflicting "GError *error" here.

We should rename the GError as we'll eventually add a bunch of macros around having a var named error around
and this functions will not work with it.